build

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Great question; I chose to not use a Makefile because, as I understand it, it's not idiomatic to use a Makefile in Go projects. I've seen this trend of using bash scripts in [larger](https://github.com/kelseyhightower/confd) projects and thought I'd give it a try.

fwiw, I'm not loving it and very well may drop back to a Makefile.